src/muya/lib/eventHandler/keyboard.js

```diff
@@ -68,6 +68,10 @@ class Keyboard {
       ) {
         return
       }
+      if (event.key === EVENT_KEYS.Control || event.key === EVENT_KEYS.Alt || event.key === EVENT_KEYS.Escape || event.key === EVENT_KEYS.Shift || event.key === EVENT_KEYS.PageUp || event.key === EVENT_KEYS.PageDown) { // we don't want to focus on the cursor if the user just hits one of these keys. Other editors do not.
```

Comment:
We should only disabled single modifiers but not PageUp and PageDown. I'm unsure whether Escape is used in this code path.

/cc @Jocs

Reply:
Sorry, should have added a note on PageUp/PageDown.  We 100% do want to move the cursor when page up/down are hit; however adding the ignore here actually improved page up/down in my tests.  The cursor still gets moved by other code, but PageUp/PageDown loses the cursor/gets stuck less without this.   I believe the refocus request every time results in less than desirable result.

As for escape, I am not sure.  Escape is used in the editor in several places and if the user's focus was not quite where they expected they may hit escape.    Looking at other editors as they don't scroll on escape so added it.

Reply:
> We 100% do want to move the cursor when page up/down are hit; however adding the ignore here actually improved page up/down in my tests.

That's what I thought first but there must be a bug that the cursor hang sometime on headings. I think finding the core issue will fix this because the page will scroll now but the cursor don't.

Reply:
> That's what I thought first but there must be a bug that the cursor hang sometime on headings. I think finding the core issue will fix this because the page will scroll now but the cursor don't.

Yeah fixing the scrolling issue in general would be best although that looked a bit more daunting than what I was going for.   

To confirm pageUp/down you want removed here?

Reply:
Yes, please. Could you open an issues for the scrolling bug?